deploy/helm/scf/assets/operations/set_opensuse_stemcells.yaml

```diff
@@ -1,17 +1,19 @@
 # This ops file sets the openSUSE stemcells.
 
-- type: replace
-  path: /stemcells/alias=default/os
-  value: opensuse-42.3
+- type: remove
+  path: /stemcells/alias=default/alias
 
 - type: replace
-  path: /stemcells/alias=default/version
-  value: 36.g03b4653-30.80-7.0.0_340.g2b599a90
+  path: /stemcells/-
+  value:
+    alias: default
+    os: ((stemcell-os))
+    version: ((stemcell-version))
 
 - type: replace
   path: /addons/name=loggregator_agent/include/stemcell/os=ubuntu-xenial/os
-  value: opensuse-42.3
+  value: default
```

Comment:
Not sure this works as intended. Wouldn't we need to:
```
- type: replace
  path: /addons/name=.../include/stemcell
  value:
    os: ...
    version: ...
```

Reply:
No, I'm just replacing which stemcell a instance group refers to; the actual stemcell definition is the previous hunk.